test/sharness/t0020-init.sh

```diff
@@ -19,7 +19,7 @@ test_expect_success "ipfs init fails" '
 	test_must_fail ipfs init 2> init_fail_out
 '
 
-test_expect_success "ipfs init output looks good" '
+test_expect_success POSIX "ipfs init output looks good" '
```

Comment:
Yeah, we already check that it fails in the above test.
On Windows the error message is different, so I could test against a different error message.